codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketVersion.java

```diff
@@ -25,49 +25,42 @@
  * </p>
  */
 public enum WebSocketVersion {
-    UNKNOWN,
 
     /**
      * <a href= "http://tools.ietf.org/html/draft-ietf-hybi-thewebsocketprotocol-00"
      * >draft-ietf-hybi-thewebsocketprotocol- 00</a>.
      */
-    V00,
+    V00("0"),
 
     /**
      * <a href= "http://tools.ietf.org/html/draft-ietf-hybi-thewebsocketprotocol-07"
      * >draft-ietf-hybi-thewebsocketprotocol- 07</a>
      */
-    V07,
+    V07("7"),
 
     /**
      * <a href= "http://tools.ietf.org/html/draft-ietf-hybi-thewebsocketprotocol-10"
      * >draft-ietf-hybi-thewebsocketprotocol- 10</a>
      */
-    V08,
+    V08("8"),
 
     /**
      * <a href="http://tools.ietf.org/html/rfc6455 ">RFC 6455</a>. This was originally <a href=
      * "http://tools.ietf.org/html/draft-ietf-hybi-thewebsocketprotocol-17" >draft-ietf-hybi-thewebsocketprotocol-
      * 17</a>
      */
-    V13;
+    V13("13");
+
+    private final String version;
+
+    WebSocketVersion(String version) {
```

Comment:
private ?

Reply:
@normanmaurer not necessary:

https://stackoverflow.com/questions/7747948/why-can-a-enum-have-a-package-private-constructor